api/src/controllers/document-controller.js

```diff
@@ -69,14 +79,7 @@ exports.protectedPost = async function (args, res, next) {
 
       // TODO: ACL is set to public-read until we know publishing rules.
       try {
-        const s3UploadResult = await s3
-          .upload({
-            Bucket: process.env.OBJECT_STORE_bucket_name,
-            Key: docResponse.key,
-            Body: args.swagger.params.upfile.value.buffer,
-            ACL: 'public-read'
-          })
-          .promise();
+        const s3UploadResult = await uploadS3Document(docResponse.key, args.swagger.params.upfile.value.buffer);
 
         s3Response = s3UploadResult;
```

Comment:
I think we can save ourselves a variable initialization here.  Just assign s3Response to the await uploadS3Document